nextjs/src/app/[locale]/publishers/contribution_page/page.jsx

```diff
@@ -180,9 +190,11 @@ export default function ContributionPage() {
               </leo-option>
             )
           })}
-          <leo-option key={'clear'} value={''}>
-            <div className='py-1'>{t('contribution_pages.clear_social')}</div>
-          </leo-option>
+          {socialLinks[category] ? (
```

Comment:
nit: would `socialLinks[category] && ` work, instead of using the ternary operator?

Reply:
yep!